WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
```

Comment:
@Laura-O, thanks for the detailed eye! Yes, this implementation (and more importantely, the [WCIF spec](https://docs.google.com/document/d/1hnzAZizTH0XyGkSYe-PxFL5xpKVWl_cvSdTzlT_kAs8/edit#heading=h.3z7evtp0blyd)) does not have support for multiple time limits per round.

Supporting multiple time limits per round feels wrong to me both as a delegate and the software team leader, so I'm not going to add support for this now.